client/src/components/advancedSelectWidget/AdvancedSelect.tsx

```diff
@@ -307,77 +314,102 @@ const AdvancedSelect = ({
-                    {(showCreateEntityComponent && (
-                      <Col md="12">
-                        {createEntityComponent(searchTerms, setDoReset)}
-                      </Col>
-                    )) || (
-                      <>
-                        <FilterAsNav
-                          items={filterDefs}
-                          currentFilter={filterType}
-                          handleOnClick={changeFilterType}
-                        />
-
-                        <FilterAsDropdown
-                          items={filterDefs}
-                          handleOnChange={handleOnChangeSelect}
-                        />
-
-                        <Col md={hasMultipleItems(filterDefs) ? 8 : 11}>
-                          <OverlayTable
-                            fieldName={fieldName}
-                            items={items}
-                            pageNum={pageNum}
-                            selectedItems={value}
-                            valueKey={valueKey}
-                            handleAddItem={item => {
-                              handleAddItem(item)
-                              if (closeOverlayOnAdd) {
-                                setDoReset(true)
-                              }
-                            }}
-                            handleRemoveItem={handleRemoveItem}
-                            objectType={objectType}
-                            columns={[""].concat(overlayColumns)}
-                            renderRow={overlayRenderRow}
-                            isLoading={isLoading}
-                            loaderMessage={
-                              <div style={{ width: "300px" }}>
-                                <div>No results found.</div>
-                                {createEntityComponent && (
-                                  <div>
-                                    <Button
-                                      id="createEntityLink"
-                                      onClick={() =>
-                                        setShowCreateEntityComponent(true)}
-                                    >
-                                      Create a new {fieldName}
-                                    </Button>
-                                  </div>
-                                )}
-                              </div>
-                            }
+                  <div
+                    style={{
+                      position: showDismiss ? "relative" : "initial",
+                      padding: showDismiss ? "10px" : "initial"
+                    }}
+                  >
+                    {showDismiss && (
+                      <BlueprintButton
+                        icon="cross"
+                        variant="minimal"
+                        onClick={() => setDoReset(true)}
+                        className={
+                          showDismiss ? BlueprintClasses.POPOVER_DISMISS : ""
+                        }
```

Comment:
`showDismiss` is always `true` here, so this can be simplified:
```javascript
                        className={BlueprintClasses.POPOVER_DISMISS}
```